homeassistant/components/zeversolar_local/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "domain": "zeversolar_local",
+  "name": "Zeversolar local",
```

Comment:
If `local` stands for using the local API, the domain and name should be zeversolar.

Reply:
It is also possible to do an integration using the remote API. So I decided to do it similar to what the solaredge integration is doing: https://www.home-assistant.io/integrations/solaredge_local/

Reply:
This integration should be named `zeversolar`, if a cloud variant is added later and they can not share the name, that one should get a `_cloud` suffix

Reply:
How is this prevented in the future ? I am not looking forward to renaming this. It is also in line with my mentioned solaredge_local.

Reply:
It's actually not in line with solaredge,.`solaredge` was added first in 0.85, and later `solaredge_local` added (in 0.95) because they could not share the domain.

Reply:
alright.

Reply:
https://github.com/home-assistant/core/pull/55440/commits/82e69a5bea3fc36b7a13d3e0413f1af221edeb92